runtime/v2/binary.go

```diff
@@ -153,6 +154,12 @@ func (b *binary) Delete(ctx context.Context) (*runtime.Exit, error) {
 	if gruntime.GOOS != "windows" && gruntime.GOOS != "freebsd" {
 		bundlePath = b.bundle.Path
 	}
+	args := []string{"-id", b.bundle.ID}
+	switch logrus.GetLevel() {
+	case logrus.DebugLevel, logrus.TraceLevel:
+		args = append(args, "-debug")
+	}
+	args = append(args, "delete")
```

Comment:
I think we're not adding the -bundle flag on the cmdline anymore for windows/freeBSD here

Reply:
Thanks, good catch! Will fix.

Reply:
Done.